goal_src/levels/maincave/mother-spider.gc

```diff
@@ -1558,7 +1558,14 @@
           (set-vector! s1-0 (sin f28-1) 0.0 (cos f28-1) 1.0)
           )
         (vector-float*! s2-1 s1-0 f30-0)
-        (dummy-35 (-> obj nav) (-> obj nav travel) (-> obj root-override trans) s2-1 s1-0 f30-0)
+        (dummy-35
+          (-> obj nav)
+          (-> obj nav travel)
+          (-> obj root-override trans)
+          s2-1
+          (the-as clip-travel-vector-to-mesh-return-info s1-0)
```

Comment:
should probably fix `mother-spider.gc` so this `s1-0` is a `clip-travel-vector-to-mesh-return-info` - otherwise there will be stack corruption when this code runs.